docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.5
+LABEL maintainer Jan Koppe <post@jankoppe.de>
+
+ARG PUMPIO__GUID=1337
+ARG PUMPIO__UID=1337
+
+ENV PUMP_LOCATION="/usr/local/pumpio"
+ENV PUMP_GIT_TAG="v4.0.0-beta.5"
+ENV PUMP_GIT_REPO="https://github.com/pump-io/pump.io.git"
+
+RUN apk add --no-cache graphicsmagick openssl nodejs python make g++ git \
+     && mkdir -p "${PUMP_LOCATION}" \
+     && git clone --recursive --depth 1 --branch "${PUMP_GIT_TAG}" "${PUMP_GIT_REPO}" "${PUMP_LOCATION}" \
```

Comment:
Yes, this is just shell. In this case the curly braces make no difference, but it's common good practice to include them, to avoid misinterpretation of the variable name.

I usually like doing the git clone style because it makes the Dockerfile more portable, but sure, you can just build this out of the repository itself with a `COPY` statement. Not sure if you would have to move the Dockerfile to the root of the repository thought.

Also, I do remember now why I did that in the first place here - I started the Dockerfile for my own use before it was suggested that I make a PR in IRC. :)